src/commands/configure.js

```diff
@@ -6,12 +6,12 @@ const { setConfig, getConfig } = require('../config')
 class ConfigureCommand extends BaseCommand {
   async run() {
     const prompts = getPrompts(['swaggerHubUrl','apiKey'])(getConfig())
-    const configFilePath = `${this.config.configDir}/config.json`
```

Comment:
Could use https://nodejs.org/api/path.html#path_path_sep to print the full path?